kubernetes/loculus/values.yaml

```diff
@@ -1183,7 +1183,7 @@ defaultOrganismConfig: &defaultOrganismConfig
   ingest: &ingest
     image: ghcr.io/loculus-project/ingest
     configFile: &ingestConfigFile
-      taxon_id: 186540
+      taxon_id: 3052460
```

Comment:
Why are you changing this? I picked a different (more basal) taxon id for ebola sudan on purpose!

https://www.ncbi.nlm.nih.gov/taxonomy/?term=186540

Reply:
We use the other taxon id for pathoplexus as well?

Reply:
you mirrored the taxon id we also use for pathoplexus

Reply:
Oh you're right, the one we used here is actually more distal, not more basal. We should use 3052460, thanks, my bad!

Reply:
https://github.com/pathoplexus/pathoplexus/blob/main/loculus_values/values.yaml#L1329

Reply:
Thanks! :)